src/main/java/com/mastery/java/task/controllers/EmployeeController.java

```diff
@@ -0,0 +1,157 @@
+package com.mastery.java.task.controllers;
+
+import com.mastery.java.task.dto.Employee;
+import com.mastery.java.task.exceptions.DuplicateEmployeeException;
+import com.mastery.java.task.exceptions.InvalidDateException;
+import com.mastery.java.task.service.EmployeeService;
+import okhttp3.*;
+import org.apache.juli.logging.Log;
+import org.apache.juli.logging.LogFactory;
+import org.apache.log4j.Level;
+import org.apache.log4j.Logger;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.jms.core.JmsMessagingTemplate;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+
+import javax.jms.Queue;
+import javax.validation.Valid;
+import java.io.IOException;
+import java.util.List;
+
+@Controller
+public class EmployeeController {
+
+    private final EmployeeService employeeService;
+    private static final Logger log = Logger.getLogger(EmployeeController.class);
+    private static final Log logCon = LogFactory.getLog(EmployeeController.class);
+    private String logLevel = "INFO";
+    private final JmsMessagingTemplate jmsMessagingTemplate;
+    private final Queue queue;
+
+    @ModelAttribute("loglevel")
+    public void addAttributeLogLevel(Model model) {
+        model.addAttribute("loglevel", logLevel);
+    }
+    @ModelAttribute("employees")
+    public void getAllEmployees(Model model){List<Employee> employees = employeeService.getAll();
+        model.addAttribute("employees", employees);}
```

Comment:
strange methods, could you explain me how should it works?

Reply:
Method must added all employees from DB in attribute "employees". My mistake, method con be easely. I changed method:

Reply:
- now it return List<Employee>
- remove Model from from the received attributes